engine/render/src/render/font.cpp

```diff
@@ -356,7 +398,34 @@ namespace dmRender
         }
         else if (FONT_GLYPH_COMPRESSION_NONE == glyph_data_compression)
         {
+            uint32_t num_out_channels;
+            switch(font_map->m_CacheFormat)
+            {
+            case dmGraphics::TEXTURE_FORMAT_LUMINANCE:  num_out_channels = 1; break;
+            case dmGraphics::TEXTURE_FORMAT_RGB:        num_out_channels = 3; break;
```

Comment:
Side-note: RGB isn't a valid format on any other API than OpenGL, which means we need to repack the data for every sub-update. We should probably deprecate that format and use RGBA instead.